catalogue/webapp/pages/PROGRESS_NOTES.md

```diff
@@ -8,26 +8,42 @@ For a more comprehensive search of all of our collections, [wellcomelibrary.org]
+### Full catalogue search
+We were previously only exposing a subset of our catalogue in search results, limited to digital and digitised materials.
+We now return results from our whole catalogue, including archives, physical works from our collections, and more.
 
-#### Fine tuning relevance
-Based on the data we already have about how search is being used, we've tested a change to the way queries are split and combined. Rather than letting results with _any_ (one or more) of the terms appear in your results (using the OR operator between query tokens), we now require that _all_ of the tokens appear in any result you see (using the AND operator between query tokens). This seems to be returning more relevant results, so we've made this the default algorithm, while kicking off a new test.
+This means you will see many more results for a given query, as we will be searching among approximately 1.2 million works, up from Approximately 350,000.
+
+To help you narrow the results our current filters should help and we have added a new search filter.
+
+### Online / In library search filter
+We've added a filter to the search form, which will allow you to choose between online and in library works.
+As there are potentially many more results being returned, this will allow you to narrow those results in another useful way.
+
+### Archive exploration
+Now that you are able to find and view archive collections, we've made some additions to the work page to help to identify and explore archives.
+  - an indication that the work is part of an archive
+  - an indication of what the parent archive is and where in its hiearchy the work being viewed sits
+  - an archive tree, which gives the ability to browse the entire archive to which the work belongs
 
 ## What's in development now
-- *Search quality*: Based on feedback from library catalogue users, we are now working to improve Title and Author searches.
-- *Archive collections*: We are bringing archive collections data into our catalogue API in order to allow users to search and browse for this material. 
+- Improvements to the archive tree to give a better experience on small screens
+- Design improvements to the archive tree
+- Improving the interface of our search form:
+  - making a clearer distinction between a catalogue and image search and explaining what they are.
+  - improving the design to allow for more filters
+  - grouping filters and other controls in more logical ways
+- Adding the ability to view items with particular access conditions on wellcomecollection.org
+- Adding the ability to request physical items on wellcomecollection.org
 
 ## What are the limits?
-- We currently only expose digitised materials, so if you're looking for a physical item from our collections, you can still use the library catalogue on [wellcomelibrary.org](https://wellcomelibrary.org).
+- *Search quality*: Based on feedback from library catalogue users, we are now working to improve Title and Author searches.
```

Comment:
We've kind of done this - known item searches are much improved now. The focus will move to adding filters, then filling out the remaining catalogue data, then we'll come back to improving broader concept searches.